packages/@aws-cdk/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -91,6 +91,13 @@ export interface ApplicationLoadBalancedFargateServiceProps extends ApplicationL
    * @default - A new security group is created.
    */
   readonly securityGroups?: ISecurityGroup[];
+
+  /**
+   * Whether to enable the ability to execute into a container.
+   *
+   * @default - undefined
+   */
+  readonly enableExecuteCommand?: boolean;
```

Comment:
I would suggest that we add this property to [`ApplicationLoadBalancedServiceBaseProps`](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts#L42) instead of here in `ApplicationLoadBalancedFargateServiceProps`. This will allow us to also add it to the `ApplicationLoadBalancedEc2Service` later.

Reply:
Good point. Could we follow with that on a separate PR when we're ready to add that functionality to `ApplicationLoadBalancedEc2Service`? I'd currently have to modify `ApplicationLoadBalancedEc2Service` to error or warn when `enableExecuteCommand` is true and I'm trying to keep this scoped to Fargate.